overlord/overlord.go

```diff
@@ -33,10 +37,14 @@ import (
 	"github.com/ubuntu-core/snappy/overlord/state"
 )
 
+var ensureInterval = 3 * time.Second
```

Comment:
as we discussed, will it make more like each 5 mins, and there will be a EnsureAfter(duration) on State to invoke it when something was changed about state that needs a new pass of Ensure soon